src/backend/api/meals.js

```diff
@@ -12,103 +13,157 @@ router.get("/", async(request, response) => {
     }
 });
 
-//Returns all meals
-router.get("/titles", async(request, response) => {
+router.get("/", async(request, response) => {
     try {
+        //console.log(meals)
         // knex syntax for selecting things. Look up the documentation for knex for further info
-        const titles = await knex("meals").select("title");
-        response.json(titles);
+        const meals = await knex("meals");
+        response.json(meals);
     } catch (error) {
         throw error;
     }
 });
-//Adds a new meal
+
 router.post("/", async(request, response) => {
     try {
-        console.log(request.body)
-        const newMeal = await knex("meals").insert(request.body);
-        response.send(`New meal: ${request.body.title}`);
+        // knex syntax for selecting things. Look up the documentation for knex for further info
+        const meals = await knex("meals")
+            .insert({
+                id: request.body.id,
```

Comment:
we need to define a method that gives an unrepeatable id 
because it is not logical to insert the id, in this case, it will be high risk for repeatable id and id should be unique
for more clarification check this solution:
https://github.com/islam-fawzy25/meal-sharing/blob/master/src/backend/api/meals.js